test/train-sets/ref/truncated_model_crash.stdout

```diff
@@ -1 +1 @@
-[critical] vw (io_buf.h:366): model load failed. Error Details: Unexpected end of file encountered., model files = model-sets/truncated_model
+[critical] vw (options_boost_po.cc:107): the argument ('-cb_adf') for option '--bit_precision' is invalid
```

Comment:
This file contains some junk in the command line which results in `-cb_adf` being a token. We know that command lines do not contain short options so we not interpret this as a value, for which it gets assigned to `--bit_precision` and fails. I believe in the past boost was interpreting this as a short option and moving forward based on that. 

The error message is bad but I believe it is more correct now. One thing I don't understand though is that for this particular test the checksum is calculated based on the invalid input, whereas if a scenario like this were to happen then most likely the checksum would not match the input and we would have a better error message.